interceptors/interceptors.go

```diff
@@ -137,6 +139,16 @@ func ServerErrorInterceptor() grpc.UnaryServerInterceptor {
 	}
 }
 
+//GRPCRecoveryInterceptor recover if server panic, return error with panic stack
+func GRPCRecoveryInterceptor() grpc.UnaryServerInterceptor {
+	opts := []grpc_recovery.Option{
+		grpc_recovery.WithRecoveryHandler(func(p interface{}) (err error) {
+			return errors.NewWithSkip(fmt.Sprintf("%v", p), 7)
```

Comment:
What's the significance of these 7 frames?

Reply:
it will list the line of wrap error,
UnaryServerInterceptor layer, 
then to the specific line of panic.
It seems we can skip the non related stack frames and directly show the panic line.

Reply:
like gin skip 3 stacks which is non-related to the panic.
https://github.com/gin-gonic/gin/blob/master/recovery.go#L54